Projects/Claims/Example/Sources/SceneDelegate.swift

```diff
@@ -0,0 +1,56 @@
+import Flow
+import Form
+import Foundation
+import UIKit
+import hCore
+import hCoreUI
+
+@available(iOS 13, *)
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+    let bag = DisposeBag()
+    var window: UIWindow?
+
+    func run() {
+        DefaultStyling.installCustom()
+
+        Localization.Locale.currentLocale = .en_SE
+
+        let navigationController = hNavigationController()
+        navigationController.navigationBar.prefersLargeTitles = true
+
+        let tapGestureRecognizer = UITapGestureRecognizer()
+        tapGestureRecognizer.numberOfTouchesRequired = 3
+
+        bag += tapGestureRecognizer.signal(forState: .recognized)
+            .onValue { _ in self.bag.dispose()
+                self.run()
+            }
+
+        window?.addGestureRecognizer(tapGestureRecognizer)
```

Comment:
I don't think you need all these, these where kind of a hack as Home couldn't be presented in a NavigationController correctly 😓